ocs_ci/ocs/ui/pvc_ui.py

```diff
@@ -51,9 +51,14 @@ def create_pvc_ui(self, sc_type, pvc_name, access_mode, pvc_size):
         logger.info("Select PVC size")
         self.do_send_keys(self.pvc_loc["pvc_size"], text=pvc_size)
 
+        logger.info(f"Select Volume Mode {vol_mode}")
```

Comment:
I think we should check OCP version here and choose the volume mode only for 4.8 version or later.

Reply:
checking OCP version now for volume mode